docs/guides/media-evidence.md

```
@@ -0,0 +1,82 @@
+# Media Evidence
+
+## Goal
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)